tools/windows_buildenv.bat

```diff
@@ -22,14 +22,14 @@ IF NOT DEFINED INSTALL_ROOT (
 
 IF DEFINED BUILDENV_RELEASE (
     SET BUILDENV_BRANCH=2.5-rel
-    SET VCPKG_TARGET_TRIPLET=x64-windows
-    SET BUILDENV_NAME=mixxx-deps-rel-2.5-x64-windows-62ca59f
-    SET BUILDENV_SHA256=e0beb24c419b14c1439c9a4dfbe58e4d0e4aafd998ea4ba5207ccda9d36c8730
+    SET VCPKG_TARGET_TRIPLET=x64-windows-release
+    SET BUILDENV_NAME=mixxx-deps-2.5-x64-windows-release-99f3970
```

Comment:
The DLLs in this release-only-buildenv are not signed. Only the release DLLs in the full buildenv `mixxx-deps-2.5-x64-windows-5d58718` are signed.